pages/common/vimdiff.md

```diff
@@ -0,0 +1,24 @@
+# vimdiff
+
+> Open up to four files in Vim and show their differences.
+> See Vim documentation for how to edit and navigate within a window.
+
+- Open files and show the differences:
+
+`vimdiff {{file1}} {{file2}}`
+
+- Use horizontal window split instead of the default vertical split:
+
+`vimdiff -o {{file1}} {{file2}}`
+
+- Close current window without saving changes:
```

Comment:
Yeah not sure about this. The moving between windows command is technically a Vim thing but I felt being able to do so as part of diffing files was key. Same if you're not too familiar with vimdiff and need to get out of there. Thoughts?